back-end/www/models/model.py

```diff
@@ -42,3 +42,56 @@ class User(db.Model):
     def __repr__(self):
         return "<User id=%r created_at=%r client_id=%r client_type=%r>" % (
                 self.id, self.created_at, self.client_id, self.client_type)
+
+
+class Location(db.Model):
+    """
+    Class representing a location.
+
+    Attributes
+    ----------
+    id : int
+        Unique identifier as primary key.
+
+    factory_id : str
+        The uuid imported from disfactory factory table.
+    
+    year: integer
+        year Marks which year the satellite photo was taken from our geo sources. 
+
+    url : string
+        URL stores the location on the map.
+
+    bbox_left_up_lat
+    bbox_left_up_lng
+    bbox_right_down_lat
+    bbox_right_down_lng:
+        The coordinates for the 2 points forming the inner boundbox for displaying the focus.        
+
+    done_at : datetime
+        The time when the location is marked done.
+
+    """
+
+    # Basic information
+    id = db.Column(db.Integer, primary_key=True)
+    factory_id = db.Column(db.String(255), nullable=False)
+    year = db.Column(db.Integer, default = 0)
```

Comment:
for consistency, should be "default=0" not "default = 0"

Reply:
and also I would say a better option is to use nullable=False and do not assign a default value to year